.github/workflows/download_onesky_translations.yml

```diff
@@ -1,36 +1,44 @@
-# This is a basic workflow to help you get started with Actions
-
 name: Download And Commit OneSky Translations
 
-# Controls when the workflow will run
 on:
-  # Triggers the workflow on push or pull request events but only for the develop branch
-  push:
-    branches: [ develop ]
-  pull_request:
-    branches: [ develop ]
+  schedule:
+    - cron: "0 23 * * *"
 
   # Allows you to run this workflow manually from the Actions tab
   workflow_dispatch:
 
-# A workflow run is made up of one or more jobs that can run sequentially or in parallel
 jobs:
-  # This workflow contains a single job called "build"
+
+  cancel-previous:
+      name: Cancel Stale In-progress Builds
+      runs-on: ubuntu-latest
+      steps:
+        - name: Cancel Previous Runs
+          uses: styfle/cancel-workflow-action@0.9.1
+          with:
+            access_token: ${{ github.token }}
+            
   build:
-    # The type of runner that the job will run on
-    runs-on: ubuntu-latest
+    runs-on: macos-11
+    env:
+      DEVELOPER_DIR: /Applications/Xcode_13.2.app/Contents/Developer
 
-    # Steps represent a sequence of tasks that will be executed as part of the job
     steps:
-      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
-      - uses: actions/checkout@v2
-
-      # Runs a single command using the runners shell
-      - name: Run a one-line script
-        run: echo Hello, world!
-
-      # Runs a set of commands using the runners shell
-      - name: Run a multi-line script
-        run: |
-          echo Add other actions to build,
-          echo test, and deploy your project.
+
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Bundle Install
+        run: bundle install
+
+      - name: Download And Commit Latest OneSky Localizations
+        env:
+          ONESKY_PUBLIC_KEY: ${{ secrets.ONESKY_PUBLIC_KEY }}
+          ONESKY_SECRET_KEY: ${{ secrets.ONESKY_SECRET_KEY }}
+        run: bundle exec fastlane cru_shared_lane_download_and_commit_latest_one_sky_localizations
+
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v3
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          title: 'Latest OneSky Translations'
```

Comment:
I would include the `branch` option for this step as well. I'd like a more descriptive branch name instead of the generic `create-pull-request/patch`

Reply:
Hey @frett changed the branch name to be more descriptive.